test/Core.Test/Utilities/StaticStoreTests.cs

```diff
@@ -28,4 +28,31 @@ public void StaticStore_GetPlan_Success(PlanType planType)
         Assert.NotNull(plan);
         Assert.Equal(planType, plan.Type);
     }
+
+    [Fact]
+    public void StaticStore_GlobalEquivalentDomains_OnlyAsciiAllowed()
+    {
+        // Ref: https://daniel.haxx.se/blog/2025/05/16/detecting-malicious-unicode/
+        // URLs can contain unicode characters that to a computer would point to completely seperate domains but to the
+        // naked eye look completely identical. For example 'g' and 'ց' look incredibly similar but when included in a 
```

Comment:
❓ The Gs are two different characters here right?

Reply:
Yep, the second one is `U+0581` want a parenthesis added with that?

Reply:
Nah all good.